tests/test_AudioClips.py

```diff
@@ -35,6 +37,23 @@ def test_audioclip():
     clip.write_audiofile(os.path.join(TMP_DIR, "audioclip.mp3"))
 
 
+def test_audioclip_io():
+    # Generate a random audio clip of 4.989 seconds at 44100 Hz,
+    # and save it to a file.
+    input_array = np.random.random((220000, 2)) * 1.98 - 0.99
+    clip = AudioArrayClip(input_array, fps=44100)
+    clip.write_audiofile(os.path.join(TMP_DIR, "random.wav"))
+    # Load the clip.
+    # The loaded clip will be slightly longer because the duration is rounded
+    # up to 4.99 seconds.
+    # Verify that the extra frames are all zero, and the remainder is identical
+    # to the original signal.
+    clip = AudioFileClip(os.path.join(TMP_DIR, "random.wav"))
+    output_array = clip.to_soundarray()
+    assert np.abs(output_array[: len(input_array)] - input_array).max() < 4e-5
```

Comment:
what about to use numpy's [assert_array_almost_equal] (https://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_array_almost_equal.html#numpy.testing.assert_array_almost_equal). 

It's the canonical way to compare arrays.  

```suggestion
    assert_array_almost_equal(output_array[:len(input_array)], input_array, decimal=4) 
```